go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v1.24.0
 	github.com/Masterminds/squirrel v1.5.4
 	github.com/ajstarks/svgo v0.0.0-20211024235047-1546f124cd8b
+	github.com/alecthomas/participle/v2 v2.1.1
```

Comment:
I used this library to simplify parsing of the filter expression. Not sure whether you have a process on how to handle new dependencies?

Reply:
As it is MIT and we have apache v2 it's alright as is.